src/mmw/js/src/analyze/templates/analyzeResults.html

```diff
@@ -1,3 +1,6 @@
 <div class="desc-region"></div>
 <div class="chart-region"></div>
 <div class="table-region"></div>
+<div class="downloadcsv-link" data-action="download-csv">
+    <i class="fa fa-download"></i> Download this data
```

Comment:
I *think* this template only gets rendered once the results have returned for a tab.